src/sdk/Enums_Internal.hpp

```diff
@@ -31591,6 +31591,7 @@ namespace app::ropeway::InputDefine {
         UI_MAP_DOWN = 140737488355328,
         UI_SORT = 281474976710656,
         UI_DISP = 562949953421312,
+        FLASHLIGHT = 1125899906842624
```

Comment:
You cannot add non-existent bindings to the game's enums. Please remove this line.

Reply:
ok removed offending line